### DISCUSSION TITLE
Amazon App Store?

### DISCUSSION THREAD

**Discussion by rdmueller** [fhdm-dev/scoppy#46] [Q&A]
This app is a great idea - thanx for building it!

Wouldn't it make sense to also deploy the app to the amazon app store in order to use cheap/old amazon fire tablets?

**Comment by fhdm-dev** [comment]
> This app is a great idea - thanx for building it!
> 
> Wouldn't it make sense to also deploy the app to the amazon app store in order to use cheap/old amazon fire tablets?

Yes, though I'm not sure how much development work would be required. I'll look into it.
